src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -4120,7 +4120,14 @@ export class StudyViewPageStore
             return _.reduce(
                 this.studyIds,
                 (acc: CancerStudy[], next) => {
-                    if (everyStudyIdToStudy[next]) {
+                    if (
+                        everyStudyIdToStudy[next] &&
+                        (!AppConfig.serverConfig
+                            .skin_show_unauthorized_studies ||
+                            (AppConfig.serverConfig
+                                .skin_show_unauthorized_studies &&
+                                everyStudyIdToStudy[next].isAuthorized == true))
```

Comment:
use ===

Reply:
== in javascript means type coercion.  it's just a good practice to use === as a rule